src/components/Login.vue

```diff
@@ -61,6 +61,8 @@
     },
     methods: {
       onLogin () {
+        window.document.body.style.overflow = 'visible'
```

Comment:
It would be better to do CSS in CSS file and not through javascript

Reply:
I totally agree but the overflow on the body is [manually set](https://github.com/kuzzleio/kuzzle-admin-console/blob/master/src/components/Materialize/Modal.vue#L60) to `hidden` when the modal is open.  
After login we push the index route in the router and the body overflow is never set to `visible` again.

Reply:
Can you at least put a comment explaining why we're changing this css property here?